cmake/modules/dts.cmake

```diff
@@ -158,6 +158,15 @@ if(DTC_OVERLAY_FILE)
     )
 endif()
 
+if(EXTRA_DTC_OVERLAY_FILE)
```

Comment:
This needs to be added to the optional variables list in the file's documentation comment above.

Reply:
together with the missing `DTC_OVERLAY_FILE` I assume 😉

Reply:
done